frontend/src/User/StoreCard.js

```diff
@@ -16,7 +16,6 @@ class StoreCard extends React.Component {
   }
 
   handleCardClick () {
-    this.props.onSub()
```

Comment:
not sure where this came from, but was breaking the user page

Reply:
@yiqiao-liu is deleting this going to be a problem?

Reply:
Are you deleting it? Yea I added this for the unit tests...